settings.gradle.dcl

```diff
@@ -0,0 +1,77 @@
+pluginManagement {
+    includeBuild("./declarative-logic")
+
+    // Check out this project as a sibling to now-in-android, that contains the declarative prototype project to use this
```

Comment:
What is this about?

Reply:
Just a note for anyone working on it who needs to make changes in the prototype at the same time, how to structure your checkouts so that you can uncomment this and have the included build work.  I can remove it from here and add a note to the README.